### ISSUE TITLE
verifying contracts that was not deployed using truffle

### ISSUE THREAD

**Issue by hellwolf** [rkalis/truffle-plugin-verify#24]
Without something like this:

```
 24       if (!(options.networkId in artifact.networks)) {
 25         artifact.networks[`${options.networkId}`] = {};
 26       }
 27       if (contractAddress) {
 28         artifact.networks[`${options.networkId}`]
 29         artifact.networks[`${options.networkId}`].address = contractAddress
 30       }
```

We would have error "Cannot set property 'address' of undefined"

**Comment by hellwolf** [comment]
```
 24       if (contractAddress) {
 25         if (!(options.networkId in artifact.networks)) {
 26           artifact.networks[`${options.networkId}`] = {};
 27         }
 28         artifact.networks[`${options.networkId}`].address = contractAddress
 29       }
 30       enforce(artifact.networks[`${options.networkId}`].address,
 31         `Cannot obtain contract address for ${contractName}`);
```

**Comment by rkalis** [comment]
Good catch, thanks! 

I don't think line 30/31 is necessary cause it's already included in line 93 (there should be no way for network.address to be undefined while the network is defined).

```js
  enforceOrThrow(
    artifact.networks && artifact.networks[`${options.networkId}`],
    `No instance of contract ${artifact.contractName} found for network id ${options.networkId}`
  )
```

Do you want to raise a PR for this, or shall I update it and push it?

**Comment by hellwolf** [comment]
Thanks for the fast response! I didn't fork the code, maybe it is easier if you could include this in your next release.

**Comment by rkalis** [comment]
Sure, will do. Thanks again!

**Comment by rkalis** [comment]
This is fixed in `v0.3.6`.
